.github/workflows/main.yml

```diff
@@ -2,11 +2,11 @@ name: Build-Android
 
 on:
   push:
-    branches: [ "main", "frontend" ]
+    branches: [ "main"]
     paths: 
       - "app/**"
   pull_request:
-    branches: [ "main", "frontend" ]
+    branches: [ "main"]
     paths: 
       - "app/**"
```

Comment:
when removing this here, this action should run on all PRs into main, regardless whether something changed in app